tools/analyzer/config.py

```
@@ -0,0 +1,122 @@
+                raise ValueError("GOOGLE_API_KEY is required for Google provider")
+            return self.google_api_key
+        elif self.provider == LLMProvider.ANTHROPIC:
+            if not self.anthropic_api_key:
+                raise ValueError("ANTHROPIC_API_KEY is required for Anthropic provider")
+            return self.anthropic_api_key
+        elif self.provider == LLMProvider.OPENAI:
+            if not self.openai_api_key:
+                raise ValueError("OPENAI_API_KEY is required for OpenAI provider")
+            return self.openai_api_key
+        else:
+            raise ValueError(f"Unknown provider: {self.provider}")
```

Pylint (3.3.7)

[refactor] Unnecessary "elif" after "return", remove the leading "el" from "elif"

(R1705)